cpp/models/abm/testing_strategy.h

```diff
@@ -88,7 +89,7 @@ class TestingCriteria
      * @param p person to be checked
      * @param l location to be checked
```

Comment:
You should add @param for Timepoint t here

Reply:
Added.